python/kserve/kserve/model_server.py

```diff
@@ -93,6 +91,8 @@ def __init__(self, http_port: int = args.http_port,
                  enable_grpc: bool = args.enable_grpc,
                  enable_docs_url: bool = args.enable_docs_url,
                  enable_latency_logging: bool = args.enable_latency_logging,
+                 configure_logging: bool = args.configure_logging,
+                 log_config: Optional[Dict] = None,
```

Comment:
Thanks @cmaddalozzo for making the change! This makes sense to me! One nit: shall we rename this `log_config` arg name to `log_config_dict` to clearly indicate the diff between this and `log_config_file`?

Reply:
Or we can align with the `UvicornServer`'s `__init__` where we just allow passing `log_config` which could either be `dict` or `str`(and defaulted to `KSERVE_LOG_CONFIG`) so that we can use only 1 argument. Also the code [here](https://github.com/kserve/kserve/pull/2884/files#diff-2109d11e056ff1c9df07471c7532d7dcab3db85d3d7c77941853af2bb426332aR116-R122) can be simplified to
```
if configure_logging:
    self.log_config = log_config
```
But I understand that might change the function signature, so the decision is yours:) IMHO, having one arg is 1)simper, 2)aligns better with `UvicornServer`.